src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -78,6 +78,9 @@ export type MatrixRTCSessionEventHandlerMap = {
  * This class doesn't deal with media at all, just membership & properties of a session.
  */
 export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, MatrixRTCSessionEventHandlerMap> {
+    // The session Id of the call, this is the call_id of the call Member event.
```

Comment:
Actually reading the react-sdk PR for context, isn't this just `callId`? ie. the discriminator used to identify different user calls in the same room? I would just call it `callId` rather than making up new terminology for it? The callIds for memberships in a session must be the same by definition, or they would belong in a different session.

Reply:
yes exactly this is just the `call_id` from the matrix rtc session.
Since the MatrixRTCSession object is very generic I felt like having call specifics in here is odd.